src/OpenLoco/Math/Trigonometry.cpp

```diff
@@ -0,0 +1,23 @@
+#include "Trigonometry.hpp"
+#include "../Interop/Interop.hpp"
+
+namespace OpenLoco::Math::Trigonometry
+{
+    // Excel Function =SIN((A1/16384)*2*PI())*32768
+    // Where A1 is 0 : 4095
+    static Interop::loco_global<int16_t[directionPrecisionHigh / 4], 0x00501B50> _quarterSine;
```

Comment:
If this is the only place this is used, perhaps we should just integrate the array?

Reply:
It is the only place so yes would be ripe for integration. Just havent done it yet.